GenericCamera/v2/_init.yaml

```diff
@@ -3,11 +3,11 @@ instances:
     ip: "127.0.0.1"
     port: 5013
     directory: "~/"
-    fileNameFormat: "{timestamp}-{index}-{total}"
+    file_name_format: "{timestamp}-{index}-{total}"
     camera: "Simulator"
-    autoExposureInterval: 60
-    minBackground: 32500.0
-    maxBackground: 33000.0
+    auto_exposure_nterval: 60
```

Comment:
`_nterval` -> `interval`, I hope.

Have you run the unit test for this package? I'm puzzled how this would have passed. I don't think this is part of the camera-specific config.

Reply:
Done.